l10n_co_geo/models/res_company.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+###############################################################################
+#    License, author and contributors information in:                         #
+#    __openerp__.py file at the root folder of this module.                   #
+###############################################################################
+
+from openerp import models, fields, api
+
+
+class ResCompany(models.Model):
+    _name = 'res.company'
+    _inherit = 'res.company'
+
+    country_id = fields.Many2one(
```

Comment:
But you have to think always globally: if you don't have city entry for other countries, you won't be able to put the country. Put an onchange, or if you wan to hard-relate both when a city entry is filled, then overwrite write and create methods for changing country_id field according that, but this second one is no usually needed, and more now with v10 version that is very simple to call new onchange methods transparently.